### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by WillemWybo** [unibe-cns/NEAT#122]
A user reported the following [issue](https://github.com/WillemWybo/NEAT/issues/1) while trying to install NEAT through pip:

>  Hi @WillemWybo, I am trying to go through your tutorial for this package.
>
> However, after looking at the link in your main repository I get this error:
>
> `(base) $ pip install neatdend`
> `ERROR: Could not find a version that satisfies the requirement neatdend (from versions: none)`
> `ERROR: No matching distribution found for neatdend`
>
> Has the package been published to pypi?

Do you know what the problem might be @jakobj ?

**Comment by jakobj** [comment]
nope, i've only published it to pypi testing (https://test.pypi.org/project/neatdend/) and pip by default doesn't check there. i can create a proper pypi package. should i do that with the recent release or do you want to create a new one?

**Comment by WillemWybo** [comment]
I've just created a release for version 0.9.1. Perhaps we upload that one as a proper pypi package?

**Comment by ZeitgeberH** [comment]
Need to install these three before installing neatdend:
 neatdend 0.9.5 depends on sphinx>=2.4.3
 neatdend 0.9.5 depends on nbsphinx>=0.5.1
 neatdend 0.9.5 depends on sphinx-gallery>=0.7

**Comment by ZeitgeberH** [comment]
Pytest after pip installation results in 13 failures
======platform linux -- Python 3.8.10, pytest-5.3.5, py-1.10.0, pluggy-0.13.1
collected 78 items

tests/test_cnet.py ....F                                                                                                             [  6%]
tests/test_compartmentfitter.py ..FFF.FFF                                                                                            [ 17%]
tests/test_compartmenttree.py .........                                                                                              [ 29%]
tests/test_greenstree.py ..                                                                                                          [ 32%]
tests/test_ionchannels.py ...                                                                                                        [ 35%]
tests/test_morphtree.py ................FF                                                                                           [ 58%]
tests/test_net.py .....                                                                                                              [ 65%]
tests/test_neurontree.py .FF....                                                                                                     [ 74%]
tests/test_phystree.py ..F.                                                                                                          [ 79%]
tests/test_sovtree.py ...                                                                                                            [ 83%]
tests/test_stree.py ..........F..                                                                                                    [100%]

**while pip install .  results in 8 failures:**
 test session starts ============================================================
platform linux -- Python 3.8.10, pytest-5.3.5, py-1.10.0, pluggy-0.13.1
collected 78 items

tests/test_cnet.py ....F                                                                                                             [  6%]
tests/test_compartmentfitter.py ..FFF.F.F                                                                                            [ 17%]
tests/test_compartmenttree.py .........                                                                                              [ 29%]
tests/test_greenstree.py ..                                                                                                          [ 32%]
tests/test_ionchannels.py ...                                                                                                        [ 35%]
tests/test_morphtree.py ..................                                                                                           [ 58%]
tests/test_net.py .....                                                                                                              [ 65%]
tests/test_neurontree.py .FF....                                                                                                     [ 74%]
tests/test_phystree.py ....                                                                                                          [ 79%]
tests/test_sovtree.py ...                                                                                                            [ 83%]
tests/test_stree.py .............

**Comment by WillemWybo** [comment]
@ZeitgeberH A lot of test failures are likely due to the lack of default ion channels compiled for neuron. Try running `compilechannels default`. 
It also seems something is wonky with the version, the latest official version is 0.9.1 and this shouldn't depend on sphinx. @jakobj I can't install `neat` from pip, when running `pip install neatdend` I get the following error:
```
ERROR: Could not find a version that satisfies the requirement neatdend (from versions: none)
ERROR: No matching distribution found for neatdend
```

**Comment by ZeitgeberH** [comment]
> 
> 
> @ZeitgeberH A lot of test failures are likely due to the lack of default ion channels compiled for neuron. Try running `compilechannels default`.
This solved all test failures! Thanks, @WillemWybo 
> It also seems something is wonky with the version, the latest official version is 0.9.1 and this shouldn't depend on sphinx. @jakobj I can't install `neat` from pip, when running `pip install neatdend` I get the following error:
> 
> ```
> ERROR: Could not find a version that satisfies the requirement neatdend (from versions: none)
> ERROR: No matching distribution found for neatdend
> ```
I was using "pip install -i https://test.pypi.org/simple/ neatdend" as shown in pypi.org website: https://test.pypi.org/project/neatdend/

**Comment by WillemWybo** [comment]
Ah thanks @ZeitgeberH , seems we still have it only as a test package, perhaps we should change this for v0.9.1 @jakobj
